### DISCUSSION TITLE
Installation process very complicated please help

### DISCUSSION THREAD

**Discussion by Unco777** [Vision-CAIR/MiniGPT-4#464] [General]
Hello guys,

I have zero knowledge in coding and I am trying to install this program.

I am somewhat stuck here:

<
Then, set the variable llama_model in the model config file to the LLM weight path.

For MiniGPT-v2, set the LLM path [here](https://github.com/Vision-CAIR/MiniGPT-4/blob/main/minigpt4/configs/models/minigpt_v2.yaml#L15) at Line 14.
>

 I downloaded this file, I manually entered the path in my home folder ( using mac sonoma), I am not sure where I need to put this file, so I just put it in my llama - 2 b chat holder.

Then

I think i am at the last step where I downloaded MiniGPT-v2 (after stage-3) ( not sure if I have to put this somewhere ) - I am using mac

Afterwards I am at this step:
For MiniGPT-v2, set the path to the pretrained checkpoint in the evaluation config file in [eval_configs/minigptv2_eval.yaml](https://github.com/Vision-CAIR/MiniGPT-4/blob/main/eval_configs/minigptv2_eval.yaml#L10) at Line 8

I downloaded the folder and I changed line 8 to where ever I put my Minigpt-v2

Finally, I think to run the program I have to use this code in my terminal: 
python demo_v2.py --cfg-path eval_configs/minigptv2_eval.yaml  --gpu-id 0

This is the error I get:

Python 3.12.1 (v3.12.1:2305ca5144, Dec  7 2023, 17:23:38) [Clang 13.0.0 (clang-1300.0.29.30)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> python demo_v2.py --cfg-path eval_configs/minigptv2_eval.yaml  --gpu-id 0
  File "<stdin>", line 1
    python demo_v2.py --cfg-path eval_configs/minigptv2_eval.yaml  --gpu-id 0
           ^^^^^^^
SyntaxError: invalid syntax

I mean I am sorry but this program is wayyyyyy to difficult to install, please help!

**Comment by JaswaniSaniya** [comment]
show complete error picture? if error still persists.
